src/benchmark_environments/util.py

```diff
@@ -1,4 +1,7 @@
 """Miscellaneous utilities."""
+from functools import partial
```

Comment:
```suggestion
import functools
```
`functools.partial` is not that much longer, and it'll be much clearer what's going on.